### ISSUE TITLE
swagger schema hardocded and references ibm properties

### ISSUE THREAD

**Issue by rabbah** [apache/incubator-openwhisk-client-go#87]
This schema should not include IBM specific properties:

See https://github.com/apache/incubator-openwhisk-client-go/blob/1759868a61729708a8ab1a6de1b409d6e2aea00a/whisk/api.go#L153-L154

I think the entire struct should be removed anyway - it overly restricts the schema unnecessarily.

**Comment by csantanapr** [comment]
Why? this limits are applicable and valid to open source gateway

**Comment by csantanapr** [comment]
They are not ibm specific

**Comment by rabbah** [comment]
The fields are called x-ibm-...

**Comment by csantanapr** [comment]
Yeah for backwards compatibility the it can x-ibm or “x- something else” don’t remember right now what’s the other option for the same

**Comment by csantanapr** [comment]
Both work in the api gateway

**Comment by rabbah** [comment]
There are no references to XConfig or XRateLimit in the project - how about we remove them... actually I'd remove the entire class, I don't think these types help much anyway and just over constrain the schema ---- or as shown several times, they get out of sync.

**Comment by rabbah** [comment]
See ongoing discussion here: https://github.com/apache/incubator-openwhisk-client-go/pull/88
